.storybook/3.COLORS.stories.mdx

```diff
@@ -1,20 +1,12 @@
 import { Meta } from '@storybook/addon-docs';
-import ActionaleMessage from '../ui/components/ui/actionable-message';
 import designTokenDiagramImage from './images/design.token.graphic.svg';
 
-<Meta title="Design Tokens / Color" />
+<Meta title="Foundations / Color" />
```

Comment:
Updating "Design Tokens" to "Foundations" as the breakpoints are technically not design tokens and I think "Foundations" is reasonably standard industry terminology for these primitive styles.